src/screens/Home/slickSettings.js

```diff
@@ -6,7 +6,7 @@ const slickSettings = {
   arrows: false,
   fade: true,
   autoplay: true,
-  speed: 500
+  speed: 500,
```

Comment:
@rafayet-monon Why we need this comma again? 😅

Reply:
Uhhh, ESLint auto fix. -_- . I am adding an eslint rule to fix all like this. @Lahphim

Reply:
Fixed in [149b5a1](https://github.com/rafayet-monon/surveyor/pull/18/commits/149b5a17fbca734b0f1aac5997d84f734bf1ceb7) @Lahphim